# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-contain/reference/quote-scoping-empty-style-boundaries-ref.html
# Commit: e37f6ffb3902
# Full Hash: e37f6ffb39025dd0b9abd9096baab78c8cb2d5fe
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-24 09:35:42
# Description:
#   Bug 1776211 - Assertion failure: aNode (aNode cannot be nullptr!), at /layout/base/nsGenConList.h:83 r=emilio
#   
#   Specifically handle empty lists in nsGenConList::BinarySearch. This
#   was hidden because the pre-existing caller also handles empty lists before
#   calling the binary search algorithm.
# ==============================================================================

diff -r abadcb426f52 -r e37f6ffb3902 testing/web-platform/tests/css/css-contain/reference/quote-scoping-empty-style-boundaries-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-contain/reference/quote-scoping-empty-style-boundaries-ref.html	Thu Jun 23 16:05:30 2022 +0000
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang=en>
+  <meta charset=utf-8>
+  <title>CSS-contain test: nested style containment and the quote element following a style boundary without any quotes</title>
+  <link rel="author" title="Martin Robinson" href="mailto:mrobinson@igalia.com">
+  <link rel=help href="https://drafts.csswg.org/css-contain-1/#containment-style">
+
+<div>
+    <div>
+        <q></q>
+    </div>
+</div>
